src/appengine/handlers/cron/cleanup.py

```diff
@@ -744,8 +744,8 @@ def _get_severity_from_labels(security_severity_label, labels):
   return data_types.SecuritySeverity.MISSING
 
 
-def _update_issue_severity_labels(policy, testcase, issue):
-  """Update severity labels on issue."""
+def _get_security_severity_issue_comment(policy, testcase, issue):
```

Comment:
nit: maybe something like _update_issue_severity_and_get_comment() to make it clear that it's updating the severity on the issue too?

Reply:
My bad, i read the function wrong and then did this rename. Fixed it now, thanks for catching.